package/MDAnalysis/coordinates/XYZ.py

```diff
@@ -408,7 +409,12 @@ def Writer(self, filename, n_atoms=None, **kwargs):
         See Also
         --------
         :class: `XYZWriter`
+
         """
+        if isinstance(atoms, AtomGroup):
```

Comment:
Could we do this instead as 
``` python
if isinstance(atom, int):
    n_atoms = atom
else:
    n_atoms = len(atom)
```
 I think eventually we want to agree on some sort of import hierarchy, so fewer internal imports make this easier